examples/managed_instance_group/main.tf

```diff
@@ -33,43 +41,76 @@ module "gce-container" {
   }
 }
 
-module "mig" {
-  source             = "GoogleCloudPlatform/managed-instance-group/google"
-  version            = "1.1.14"
-  project            = var.project_id
-  region             = var.region
-  zone               = var.zone
-  name               = var.mig_name
-  machine_type       = var.machine_type
-  compute_image      = module.gce-container.source_image
-  size               = var.mig_instance_count
-  service_port       = var.image_port
-  service_port_name  = "http"
-  http_health_check  = "true"
-  subnetwork         = var.subnetwork
-  subnetwork_project = var.subnetwork_project
-  ssh_source_ranges  = ["0.0.0.0/0"]
-  target_tags        = ["container-vm-test-mig"]
-
-  metadata = merge(var.additional_metadata, map("gce-container-declaration", module.gce-container.metadata_value))
-
-  instance_labels = {
+resource "google_compute_network" "default" {
+  name                    = var.network
+  auto_create_subnetworks = "false"
+}
+
+resource "google_compute_subnetwork" "default" {
+  name                     = var.network
+  ip_cidr_range            = "10.125.0.0/20"
+  network                  = google_compute_network.default.self_link
+  region                   = var.region
+  private_ip_google_access = true
+}
+
+# Router and Cloud NAT are required for installing packages from repos (apache, php etc)
+resource "google_compute_router" "default" {
+  name    = "${var.network}-gw-group1"
+  network = google_compute_network.default.self_link
+  region  = var.region
+}
+
+module "cloud-nat" {
+  source     = "terraform-google-modules/cloud-nat/google"
+  version    = "1.0.0"
+  router     = google_compute_router.default.name
+  project_id = var.project_id
+  region     = var.region
+  name       = "${var.network}-cloud-nat-group1"
+}
+
+module "mig_template" {
+  source               = "terraform-google-modules/vm/google//modules/instance_template"
+  version              = "1.0.0"
+  network              = google_compute_network.default.self_link
+  subnetwork           = google_compute_subnetwork.default.self_link
+  service_account      = var.service_account
+  name_prefix          = var.network
+  source_image_family  = "cos-stable"
+  source_image_project = "cos-cloud"
+  source_image         = reverse(split("/", module.gce-container.source_image))[0]
+  metadata             = merge(var.additional_metadata, map("gce-container-declaration", module.gce-container.metadata_value))
+  tags                 = [
+    "container-vm-test-mig"]
```

Comment:
See above RE: moving trailing `]` to separate line

Reply:
@kopachevsky @Jberlinsky 
[Fixed](https://github.com/terraform-google-modules/terraform-google-container-vm/compare/21506b0e8d88a8bee72d02dc63094b8276ccd288..b6d0f962d7a2074d4392d31a3368b8c6808df77b)